leshan-client-demo/src/main/java/org/eclipse/leshan/client/demo/LeshanClientDemo.java

```diff
@@ -135,6 +130,20 @@ public static void main(final String[] args) {
         options.addOption("scert", true,
                 "The path to your server certificate file.\n The certificate should be in X509v3 format (DER encoding).");
         options.addOption("oscore", false, "Use OSCORE for communication between client and server.");
```

Comment:
Do we still need this global option `oscore` ? or maybe just the presence of new oscore option is enough ?

Reply:
True, we do not need that. Instead we can check if the OSCORE master secret is set. I removed it in commit cf7aa439ab91a3b367945217ef38d849c402adc6